rpcd/playbooks/roles/rpc_maas/templates/galera_check.yaml.j2

```diff
@@ -34,3 +34,65 @@ alarms      :
             if (percentage(metric["mysql_current_connections"], metric["mysql_max_configured_connections"]) > {{ mysql_connection_warning_threshold }} ) {
                 return new AlarmStatus(WARNING, "Mysql connectons has exceeded {{ mysql_connection_warning_threshold }}% of configured maximum");
             }
+    mysql_max_used_connections :
+        label                   : mysql_max_used_connections--{{ ansible_hostname }}
+        notification_plan_id    : "{{ maas_notification_plan }}"
+        criteria                : |
+            :set consecutiveCount={{ maas_alarm_local_consecutive_count }}
+            if (metric["mysql_max_seen_connections"] > metric["mysql_max_configured_connections"]) {
```

Comment:
Still I think you're right that Max_used_connections can never be higher than max_connections. 
I'll do a percentage and check against a threshold then submit another commit shortly.